projboard/tests/test_user.py

```diff
@@ -10,28 +10,26 @@
 
 @pytest.fixture
 def generate_user():
-    user1 = User.create_user(name="RAWAD", nickname="R1998", email="rawad@gmail.com", password="rawad!@#")
-    user2 = User.create_user(name="John", nickname="J99", email="John@gmail.com", password="John123")
+    user1 = User.get_user(name="RAWAD", nickname="R1998", email="rawad@gmail.com", password="rawad!@#")
     user1.save()
-    user2.save()
-    return [user1, user2]
+    return user1
 
 
 @pytest.mark.django_db
 class TestUserModel:
     def test_create_user(self, generate_user):
-        user = generate_user[0]
+        user = generate_user
```

Comment:
I think the tests should search it in database, isnt it?

Reply:
@Yarboa You mean to some think like that:
```
@pytest.mark.django_db
def test_create_user(self, generate_user):
        assert generate_user in User.objects.all()
```

?

Reply:
This could be one option, the other query a method with user fixture name,
retrieve from db, assert they are equal

Reply:
@Yarboa Can you review now please?

Reply:
Almost 
   @pytest.mark.django_db()
    def test_create_user(self, generate_user):
        generate_user.save()
        assert generate_user in User.objects.all()
        
 Please move the save to fixture